modules/assim.batch/vignettes/AssimBatchVignette.Rmd

```diff
@@ -194,23 +211,26 @@ If you are using methods other than *bruteforce* and *bruteforce.bs*, some addit
 * DEzs : Differential Evolution with a snooker updater
 * DREAM : Differential Evolution Adaptive Metropolis 
 * DREAMzs : Differential Evolution Adaptive Metropolis with a snooker updater
+* Twalk : "traverse" or "thoughtful" walk, a general purpose sampling algorithm
 * SMC : Sequential Monte Carlo
 
 The name of the chosen sampler would be passed under `<sampler>` tag within the `<bt.settings>` block :
 
+It is not possible to use some of the samplers in this package for univariate cases. The ones that you can use for univariate cases are: "Metropolis", "DE", "DEzs" and "Twalk".
```

Comment:
I'll work on BayesianTools code and usage some more later. I'd like to implement the parallelization